### PR TITLE
Upload autodistill classification notebook

### PR DESCRIPTION
**Pull request by capjamesg**
This PR adds the accompanying notebook for our [Train an Image Classification Model with No Labeling](https://blog.roboflow.com/train-classification-model-no-labeling/) tutorial.

### CODE REVIEW

**Comment by SkalskiP** [comment]
Hi @capjamesg 👋🏻!

Unfortunately, I need to ask you to make a few changes. 

- The whole notebook is 80000+ lines long, but most of those lines are entirely redundant. Take a look [here](https://github.com/roboflow/notebooks/blob/upload-autodistill-notebook/notebooks/how-to-train-yolov8-classification-no-labeling.ipynb). Output of call `[5]` makes it super hard to follow the notebook. Please assign `base_model.label` to a variable or remove the `[5]` output before committing the notebook.
- Don't use paths like `"/content/dataset/"` or `/content/dataset/valid/damaged sign/IMG_8113_jpg.rf.2873a7e5ccb38fcc5c31dc7f20adeae0.jpg"` as those won't work for anyone running the notebook locally, on Kaggle or AWS.

**Comment by capjamesg** [comment]
@SkalskiP I have responded to your feedback in my latest commits.

**Comment by SkalskiP** [comment]
@capjamesg I tried to run the notebook and it doesn't work. 

Screenshot 2023-07-05 at 12 25 17

**Comment by capjamesg** [comment]
Ah, replace `train2` with `train`. I will submit a PR for that. I trained multiple different models in that notebook.

**Comment by SkalskiP** [comment]
@capjamesg please submit PR with working notebook.

**Comment by capjamesg** [comment]
I accidentally committed a change from another branch on this branch so I am closing this PR. The new PR, which includes the requested changes, is #168.

**Comment by SkalskiP** [comment]
Okey :)
